docs/reference/modules/http.asciidoc

```diff
@@ -13,105 +14,118 @@ http://en.wikipedia.org/wiki/Keepalive#HTTP_Keepalive[HTTP keep alive]
++
+IMPORTANT: A wildcard (`*`) is a valid value but is considered a security risk, as your {es} instance is open to cross origin requests from *anywhere*.
 
-|`http.cors.max-age` |Browsers send a "preflight" OPTIONS-request to
-determine CORS settings. `max-age` defines how long the result should
-be cached for. Defaults to `1728000` (20 days)
+// end::http-cors-allow-origin-tag[]
 
-|`http.cors.allow-methods` |Which methods to allow. Defaults to
-`OPTIONS, HEAD, GET, POST, PUT, DELETE`.
+[[http-cors-max-age]]
+// tag::http-cors-max-age-tag[]
+`http.cors.max-age` {ess-icon}::
+Browsers send a "preflight" OPTIONS-request to determine CORS settings. `max-age` defines how long the result should be cached for. Defaults to `1728000` (20 days).
+// end::http-cors-max-age-tag[]
 
-|`http.cors.allow-headers` |Which headers to allow. Defaults to
-`X-Requested-With, Content-Type, Content-Length`.
+[[http-cors-allow-methods]]
+// tag::http-cors-allow-methods-tag[]
+`http.cors.allow-methods` {ess-icon}::
+Which methods to allow. Defaults to `OPTIONS, HEAD, GET, POST, PUT, DELETE`.
+// end::http-cors-allow-methods-tag[]
 
-|`http.cors.allow-credentials` | Whether the `Access-Control-Allow-Credentials`
-header should be returned. Note: This header is only returned, when the setting is
-set to `true`. Defaults to `false`
+[[http-cors-allow-headers]]
+// tag::http-cors-allow-headers-tag[]
+`http.cors.allow-headers` {ess-icon}::
+Which headers to allow. Defaults to `X-Requested-With, Content-Type, Content-Length`.
+// end::http-cors-allow-headers-tag[]
 
-|`http.detailed_errors.enabled` |Enables or disables the output of detailed error messages
-and stack traces in response output. Note: When set to `false` and the `error_trace` request
-parameter is specified, an error will be returned; when `error_trace` is not specified, a
-simple message will be returned. Defaults to `true`
+[[http-cors-allow-credentials]]
+// tag::http-cors-allow-credentials-tag[]
+`http.cors.allow-credentials` {ess-icon}::
+Whether the `Access-Control-Allow-Credentials` header should be returned. Defaults to `false`.
++
+NOTE: This header is only returned when the setting is set to `true`.
 
-|`http.pipelining.max_events` |The maximum number of events to be queued up in memory before an HTTP connection is closed, defaults to `10000`.
+// end::http-cors-allow-credentials-tag[]
 
-|`http.max_warning_header_count` |The maximum number of warning headers in
- client HTTP responses, defaults to unbounded.
+`http.detailed_errors.enabled`::
+Enables or disables the output of detailed error messages and stack traces in response output. Defaults to `true`.
++
+NOTE: When set to `false` and the `error_trace` request
```

Comment:
I think that this revision is more clear:

>If `true`, enables the output of detailed error messages and stack traces in the response output. Defaults to `true`.

>If `false`, use the `error_trace` parameter to enable stack traces and return detailed error messages. Otherwise, only a simple message will be returned.